Anasazi/RBGen compilation error

This morning a Trilinos compilation error cropped up in Albany nightly testing having to do with Anasazi/RBen:

```
repos/Trilinos/packages/anasazi/epetra/src-rbgen/../../src-rbgen/RBGen_Filter.hpp:205:46: error: ‘tval’ is not captured
```

(or see here http://cdash.sandia.gov/CDash-2-3-0/viewBuildError.php?buildid=76890 ).  Can someone please fix this issue?

@trilinos/anasazi

--- a/packages/anasazi/src-rbgen/RBGen_Filter.hpp
+++ b/packages/anasazi/src-rbgen/RBGen_Filter.hpp
@@ -201,7 +201,7 @@ namespace RBGen {
       std::vector<int> ret;
       if (LARGEST == which_) {
         const int num = find(svals.begin(),svals.end(),
-                             [] (const ScalarType sval) {
+                             [=] (const ScalarType sval) {
                                return sval < tval;
                              }) - svals.begin();
         ret.resize(num);
@@ -211,7 +211,7 @@ namespace RBGen {
       }
       else if (SMALLEST == which_) {
         const int num = svals.end() - find(svals.begin(),svals.end(),
-                                           [] (const ScalarType sval) {
+                                           [=] (const ScalarType sval) {
                                              return sval < tval;
                                            }) + 1;
         ret.resize(num);

